addons/binding/org.openhab.binding.rfxcom/src/main/java/org/openhab/binding/rfxcom/internal/messages/RFXComInterfaceControlMessage.java

```diff
@@ -0,0 +1,126 @@
+            | (configuration.enableRSL              ? 0x10 : 0x00)
+            | (configuration.enableLighting4        ? 0x08 : 0x00)
+            | (configuration.enableFineOffsetViking ? 0x04 : 0x00)
+            | (configuration.enableRubicson         ? 0x02 : 0x00)
+            | (configuration.enableAEBlyss          ? 0x01 : 0x00)
+        );
+
+        data[8] = (byte)(
+              (configuration.enableBlindsT1T2T3T4   ? 0x80 : 0x00)
+            | (configuration.enableBlindsT0         ? 0x40 : 0x00)
+            | (configuration.enableProGuard         ? 0x20 : 0x00)
+         // | (configuration.enableFS20Packets      ? 0x10 : 0x00)
+            | (configuration.enableLaCrosse         ? 0x08 : 0x00)
+            | (configuration.enableHidekiUPM        ? 0x04 : 0x00)
+            | (configuration.enableADLightwaveRF    ? 0x02 : 0x00)
+            | (configuration.enableMertik           ? 0x01 : 0x00)
+        );
+
+        data[9] = (byte)(
+              (configuration.enableVisonic          ? 0x80 : 0x00)
+            | (configuration.enableATI              ? 0x40 : 0x00)
+            | (configuration.enableOregonScientific ? 0x20 : 0x00)
+            | (configuration.enableMeiantech        ? 0x10 : 0x00)
+            | (configuration.enableHomeEasyEU       ? 0x08 : 0x00)
+            | (configuration.enableAC               ? 0x04 : 0x00)
+            | (configuration.enableARC              ? 0x02 : 0x00)
+            | (configuration.enableX10              ? 0x01 : 0x00)
+        );
+
+        data[10] = 0;
+        data[11] = 0;
+        data[12] = 0;
+        data[13] = 0;
+    }
+
+    public RFXComInterfaceControlMessage(byte[] data) throws RFXComException {
+        // We should never receive control messages
+        throw new RFXComException("Not supported");
+    }
+
+    @Override
+    public byte[] decodeMessage() {
+        return data;
+    }
+
+    @Override
+    public void encodeMessage(byte[] data) throws RFXComException {
+        throw new RFXComException("Not supported");
+    }
+
+    @Override
+    public State convertToState(RFXComValueSelector valueSelector) throws RFXComException {
+        throw new RFXComException("Not supported");
+    }
+
+    @Override
+    public void setSubType(Object subType) throws RFXComException {
+        throw new RFXComException("Not supported");
+    }
+
+    @Override
+    public void setDeviceId(String deviceId) throws RFXComException {
+        throw new RFXComException("Not supported");
+    }
+
+    @Override
+    public void convertFromState(RFXComValueSelector valueSelector, Type type) throws RFXComException {
+        throw new RFXComException("Not supported");
+    }
+
+    @Override
+    public Object convertSubType(String subType) throws RFXComException {
+        throw new RFXComException("Not supported");
+    }
+
+    @Override
+    public List<RFXComValueSelector> getSupportedInputValueSelectors() throws RFXComException {
+        return null;
```

Comment:
There's no one to help :-). It's only called from RFXComHandler.onDeviceMessageReceived and this is an outgoing only message. It could exec("global_thermonuclear_war") for all it matters.